### PR TITLE
Testing PHP 7.2 and 7.3

### PR DESCRIPTION
**Pull request by dannylamb**
**JIRA Ticket**: https://github.com/Islandora/documentation/issues/1267

# What does this Pull Request do?

Tests on 7.2 and 7.3, which we should do everywhere.  Hence I'm adding this as part of the CLAWdit.

# Interested parties
@Islandora/8-x-committers

### CODE REVIEW

**Comment by dannylamb** [comment]
Hrm... not ingoring the 7.3 fail for some reason....

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 php:
-  - 7.1
   - 7.2
-services:
-  - mysql
+  - 7.3
 
 matrix:
     fast_finish: true
+    allowed_failures:
```
Comment (whikloj) [inline .travis.yml]: `allow_failures:`

Comment (dannylamb) [reply .travis.yml]: :man_facepalming:

Comment (dannylamb) [reply .travis.yml]: i bet  i copy/pasted that across every repo...

Comment (dannylamb) [reply .travis.yml]: :sob:

Comment (whikloj) [reply .travis.yml]: 😿

**Pull request merged**